apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx

```diff
@@ -0,0 +1,35 @@
+import { SjfMetaData } from '../../../../schemas/patientSchema'
+import { useAddVardenhetMutation } from '../../../../store/api'
+import { OpenInformationCard } from './OpenInformationCard'
+import { BlockedInformationCard } from './BlockedInformationCard'
+
+export function PatientOverview({ sjfMetaData, patientId }: { sjfMetaData: SjfMetaData | undefined; patientId: string }) {
+  const [addUnit] = useAddVardenhetMutation()
+
+  if (!sjfMetaData || !patientId) {
+    return null
+  }
+
+  const handleGetCareUnitInformation = (id: string) => {
+    addUnit({ patientId, vardenhetId: id })
+  }
+
+  return (
+    <div className="flex flex-col gap-5">
+      <OpenInformationCard
+        title="Ospärrad information inom vårdgivare"
+        subTitle="Vårdenhet att hämta information från"
+        description="Det finns ospärrad information hos en annan vårdenhet inom din vårdgivare. Du kan klicka nedan för att visa vilka vårdenheter som
+        har denna information och få möjlighet att inhämta den."
```

Comment:
These attributes are unique for; and could be moved to `OpenInformationCard`

Reply:
I'll see if I can move it in the next jira but I think for reuse purposes it should stay like this for now.